src/common/pf_lldp.c

```diff
@@ -993,17 +981,19 @@ static void pf_lldp_send_port (pnet_t * net, int loc_port_num)
    }
 }
 
-void pf_lldp_send (pnet_t * net)
+void pf_lldp_send (pnet_t * net, int loc_port_number)
 {
-   pf_lldp_send_port (net, PNET_PORT_1);
+   pf_lldp_send_port (net, loc_port_number);
```

Comment:
You can remove one of `pf_lldp_send_port()` and `pf_lldp_send()`. My intention was to call `pf_lldp_send()` by timer, and it would figure out which ports to send to. 
Now you use individal LLDP send timers, so it is no longer required.

Reply:
Done